src/components/BusinessesMap/example.jsx

```diff
@@ -0,0 +1,103 @@
+const { compose, withProps, lifecycle } = require('recompose');
+const {
+    withScriptjs,
+    withGoogleMap,
+    GoogleMap,
+    Marker,
+} = require('react-google-maps');
+const { SearchBox } = require('react-google-maps/lib/components/places/SearchBox');
+
+const MapWithASearchBox = compose(
+    withProps({
+        googleMapURL: 'https://maps.googleapis.com/maps/api/js?key=AIzaSyC4R6AN7SmujjPUIGKdyao2Kqitzr1kiRg&v=3.exp&libraries=geometry,drawing,places',
```

Comment:
not a very big deal since the the api_key can be found in the browser, but let's keep the code consistent by keeping the key as an environment variable

Reply:
Oh sorry this file will be deleted, it is an example I got from the documentation to help who ever is working on it, but its very different from what im using i will delete it.